Formula/opencoarrays.rb

```diff
@@ -12,14 +12,20 @@ class Opencoarrays < Formula
     sha256 "84cbce876804c2f46cda80bec9b4b310aa0d06300e65473a89207ae6ea8679a3" => :el_capitan
   end
 
+  option "without-failed-image-support", "Disable F2018 failed image support that uses experimental MPI features"
```

Comment:
Options mean that users have to compile from source, so we want to avoid them as much as possible. Experimental MPI support (or its absence) should be autodetected by cmake.